router/voteCallbacks.go

```diff
@@ -73,6 +74,37 @@ func (r *Router) submitEnvelope(request routerRequest) {
 	}
 }
 
+func (r *Router) getStats(request routerRequest) {
+	stats := new(models.VochainStats)
+
+	stats.BlockHeight = uint32(r.vocapp.State.Header(true).Height)
+	stats.BlockTimeStamp = int32(r.vocapp.State.Header(true).Timestamp)
+
+	stats.EntityCount = r.Scrutinizer.EntityCount()
+
+	votes, err := r.Scrutinizer.GetEnvelopeHeight([]byte{})
+	if err != nil {
+		log.Warnf("could not count vote envelopes: %s", err)
+	}
+	stats.EnvelopeCount = votes
+
+	stats.ProcessCount = r.Scrutinizer.ProcessCount([]byte{})
+	vals, _ := r.vocapp.State.Validators(true)
+	stats.ValidatorCount = int32(len(vals))
+	stats.BlockTime = r.vocinfo.BlockTimes()[:]
+
+	stats.ChainID = r.vocapp.Node.GenesisDoc().ChainID
```

Comment:
Lets store this in the vocapp directly on app.go NewBaseApplication constructor and make a function to export it `ChainID()`

Reply:
The Node isn't assigned in the App constructor, it has to be done at a second step. So we don't have access to the ChainID during app construction. We could create a SetNode method, though, which assigns the App.Node and also the App.ChainID

Reply:
SGTM